server/repositories/browserscope.js

```diff
@@ -22,10 +22,14 @@ exports.register = function (server, options, next) {
           try {
             resolve(JSON.parse(str).test_key);
           } catch (e) {
-            reject(new Error('Unexpected response from browserscope.org'));
+            server.log('error', e);
```

Comment:
I've been writing a lot of Go lately which has me in the short variable name habbit 😸 

> Variable names in Go should be short rather than long. This is especially true for local variables with limited scope. Prefer c to lineCount. Prefer i to sliceIndex.
> – [Go Code Review](https://github.com/golang/go/wiki/CodeReviewComments#variable-names)

Looks like [ESLint could help](http://eslint.org/docs/rules/id-length) me when writing JS